src/lib/dialog/dialog.md

```diff
@@ -29,6 +29,37 @@ Components created via `MdDialog` can _inject_ `MdDialogRef` and use it to close
 in which they are contained. When closing, an optional result value can be provided. This result
 value is forwarded as the result of the `afterClosed` promise. 
 
+### Sharing Data with the Dialog component.
+Depending on what you are doing you might want to share data with your dialog component. The dialog component has a data option that you can include in order to pass data from the open to the dialog. 
+
+Passing outside data to your component is as simple as.
+```ts
+import {MdDialog} from '@angular/material';
```

Comment:
In this example it's pretty obvious where `MdDialog` comes from so you can remove this line and the `private dialog: MdDialog`.